tests/Integration/Infrastructure/Controller/Regulation/Fragments/AddMeasureControllerTest.php

```diff
@@ -159,9 +159,9 @@ public function testAdd(): void
 
         $crawler = $client->request($form->getMethod(), $form->getUri(), $values, $form->getPhpFiles());
 
-        $this->assertResponseStatusCodeSame(200);
+        $this->assertResponseStatusCodeSame(expectedCode: 200);
```

Comment:
Mieux vaut éviter ces changements stylistiques intempestifs ? (J'imagine que ça vient des annotations VSCode ?)

```suggestion
        $this->assertResponseStatusCodeSame(200);
```